src/main/java/ua/lviv/iot/java/labs/forth/Main.java

```diff
@@ -0,0 +1,8 @@
+package ua.lviv.iot.java.labs.forth;
+
+public class Main {
```

Comment:
it's better not to name class Main, Application will be better

Reply:
Done